extended-tests/build.gradle.kts

```diff
@@ -14,7 +14,8 @@ dependencies {
   testImplementation(libs.okhttp3.mockserver)
   testImplementation(libs.assertj.core)
   testImplementation("org.springframework.boot:spring-boot-starter-web")
-
+  testImplementation(libs.kotlin.serialization.json)
```

Comment:
Why wasn't this required before this change?

Reply:
There is a dependency conflict with the Wallet SDK. This is to force the dependency version.